src/main/java/net/syscon/elite/service/transformers/CourtCaseTransformer.java

```diff
@@ -0,0 +1,39 @@
+package net.syscon.elite.service.transformers;
+
+import net.syscon.elite.api.model.Agency;
+import net.syscon.elite.api.model.CourtCase;
+import net.syscon.elite.repository.jpa.model.CaseStatus;
+import net.syscon.elite.repository.jpa.model.LegalCaseType;
+import net.syscon.elite.repository.jpa.model.OffenderCourtCase;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * Simple transform object to help reduce boiler plate mapping of entity {@link OffenderCourtCase} to model object {@link CourtCase}.
+ */
+public class CourtCaseTransformer {
+
+    public static CourtCase transform(final OffenderCourtCase courtCase) {
+        return CourtCase.builder()
+                .id(courtCase.getId())
+                .caseSeq(courtCase.getCaseSeq())
+                .beginDate(courtCase.getBeginDate())
+                .agency(Agency.builder()
+                        .agencyId(courtCase.getAgencyLocation().getId())
+                        .agencyType(courtCase.getAgencyLocation().getType())
+                        .active(courtCase.getAgencyLocation().getActiveFlag().isActive())
+                        .description(courtCase.getAgencyLocation().getDescription())
+                        .build())
```

Comment:
Shouldn't Agency have its own transformer?

Reply:
No keeping people happy here ;)

Indeed it should this is just the first.  There be one and also one for court events....